src/Services/ItemListService.php

```diff
@@ -183,6 +185,12 @@ public function getItemList(
             $searchFactory->withCategories();
         }
 
+        if ($withVariationProperties) {
+            $searchFactory->withResultFields(
+                array_merge($searchFactory->getResultFields(), ['variationProperties.*'])
```

Comment:
Warum kannst du `'variationProperties.*'` nicht in deiner ResultFields .json-Datei ergänzen?

Reply:
weil das dann fuer alle requests diese daten pullt. wollen wir das?